src/Facilitator.ts

```diff
@@ -73,32 +29,10 @@ export default class Facilitator {
    */
   public async stop(): Promise<void> {
     if (this.originSubscriber) {
```

Comment:
as now `this.originSubscriber` and `this.auxiliarySubscriber` are mandatory properties, we can remove the if condition